.gitignore

```diff
@@ -2,3 +2,4 @@ package-lock.json
 *egg-info
 .vscode
 .DS_Store
+CHANGELOG
```

Comment:
I'm curious why we're ignoring CHANGELOG here. I didn't see and specific callout in the PR or the commits or Issues that I could find. There doesn't appear to be a CHANGELOG file today so I'm curious why we're specifically ignoring it.

Reply:
Ah, that was something I did early on and forgot to cleanup.  My bad.